src/Plugins/ComplexCore/test/ApproximatePointCloudHullTest.cpp

```diff
@@ -1,62 +1,126 @@
 
 #include <catch2/catch.hpp>
 
-#include "complex/UnitTest/UnitTestCommon.hpp"
-
 #include "ComplexCore/ComplexCore_test_dirs.hpp"
 #include "ComplexCore/Filters/ApproximatePointCloudHull.hpp"
+#include "ComplexCore/Filters/StlFileReaderFilter.hpp"
+#include "complex/DataStructure/Geometry/TriangleGeom.hpp"
+#include "complex/DataStructure/Geometry/VertexGeom.hpp"
+#include "complex/Filter/Actions/CreateVertexGeometryAction.hpp"
+#include "complex/Parameters/FileSystemPathParameter.hpp"
+#include "complex/UnitTest/UnitTestCommon.hpp"
+#include "complex/Utilities/Parsing/HDF5/H5FileWriter.hpp"
 
 using namespace complex;
 using namespace complex::Constants;
 
+static std::vector<float> s_Vertices = {0.0F,      0.0F,       -8.305164E-7F,  40.35763F, 0.0F,       -8.305164E-7F,  139.6424F, 0.0F,       -8.305164E-7F,  180.0F,    0.0F,       -8.305164E-7F,
+                                        45.55068F, 1.3076F,    -7.733594E-7F,  134.4493F, 1.3076F,    -7.733594E-7F,  50.82293F, 2.2462597F, -7.323292E-7F,  56.14818F, 2.8113308F, -7.076292E-7F,
+                                        123.8518F, 2.8113308F, -7.076292E-7F,  129.1771F, 2.2462597F, -7.323292E-7F,  61.5F,     3.0F,       -6.993822E-7F,  118.5F,    3.0F,       -6.993822E-7F,
+                                        50.82293F, 16.753738F, -9.8187236E-8F, 56.14818F, 16.18867F,  -1.2288719E-7F, 61.5F,     16.0F,      -1.3113416E-7F, 118.5F,    16.0F,      -1.3113416E-7F,
+                                        123.8518F, 16.18867F,  -1.2288719E-7F, 129.1771F, 16.753738F, -9.8187236E-8F, 45.55068F, 17.692402F, -5.7156925E-8F, 134.4493F, 17.692402F, -5.7156925E-8F,
+                                        0.0F,      19.0F,      0.0F,           40.35763F, 19.0F,      0.0F,           139.6424F, 19.0F,      0.0F,           180.0F,    19.0F,      0.0F,
+                                        0.0F,      0.0F,       3.1999993F,     40.35763F, 0.0F,       3.1999993F,     139.6424F, 0.0F,       3.1999993F,     180.0F,    0.0F,       3.1999993F,
+                                        45.55068F, 1.3076F,    3.1999993F,     134.4493F, 1.3076F,    3.1999993F,     50.82293F, 2.2462597F, 3.1999993F,     56.14818F, 2.8113308F, 3.1999993F,
+                                        123.8518F, 2.8113308F, 3.1999993F,     129.1771F, 2.2462597F, 3.1999993F,     61.5F,     3.0F,       3.1999993F,     118.5F,    3.0F,       3.1999993F,
+                                        50.82293F, 16.753738F, 3.2F,           56.14818F, 16.18867F,  3.1999998F,     61.5F,     16.0F,      3.1999998F,     118.5F,    16.0F,      3.1999998F,
+                                        123.8518F, 16.18867F,  3.1999998F,     129.1771F, 16.753738F, 3.2F,           45.55068F, 17.692402F, 3.2F,           134.4493F, 17.692402F, 3.2F,
+                                        0.0F,      19.0F,      3.2F,           40.35763F, 19.0F,      3.2F,           139.6424F, 19.0F,      3.2F,           180.0F,    19.0F,      3.2};
```

Comment:
```suggestion
namespace
{
static const std::vector<float32> s_Vertices = {
    0.0F,      0.0F,       -8.305164E-7F,  40.35763F, 0.0F,       -8.305164E-7F,  139.6424F, 0.0F,       -8.305164E-7F,  180.0F,    0.0F,       -8.305164E-7F,
    45.55068F, 1.3076F,    -7.733594E-7F,  134.4493F, 1.3076F,    -7.733594E-7F,  50.82293F, 2.2462597F, -7.323292E-7F,  56.14818F, 2.8113308F, -7.076292E-7F,
    123.8518F, 2.8113308F, -7.076292E-7F,  129.1771F, 2.2462597F, -7.323292E-7F,  61.5F,     3.0F,       -6.993822E-7F,  118.5F,    3.0F,       -6.993822E-7F,
    50.82293F, 16.753738F, -9.8187236E-8F, 56.14818F, 16.18867F,  -1.2288719E-7F, 61.5F,     16.0F,      -1.3113416E-7F, 118.5F,    16.0F,      -1.3113416E-7F,
    123.8518F, 16.18867F,  -1.2288719E-7F, 129.1771F, 16.753738F, -9.8187236E-8F, 45.55068F, 17.692402F, -5.7156925E-8F, 134.4493F, 17.692402F, -5.7156925E-8F,
    0.0F,      19.0F,      0.0F,           40.35763F, 19.0F,      0.0F,           139.6424F, 19.0F,      0.0F,           180.0F,    19.0F,      0.0F,
    0.0F,      0.0F,       3.1999993F,     40.35763F, 0.0F,       3.1999993F,     139.6424F, 0.0F,       3.1999993F,     180.0F,    0.0F,       3.1999993F,
    45.55068F, 1.3076F,    3.1999993F,     134.4493F, 1.3076F,    3.1999993F,     50.82293F, 2.2462597F, 3.1999993F,     56.14818F, 2.8113308F, 3.1999993F,
    123.8518F, 2.8113308F, 3.1999993F,     129.1771F, 2.2462597F, 3.1999993F,     61.5F,     3.0F,       3.1999993F,     118.5F,    3.0F,       3.1999993F,
    50.82293F, 16.753738F, 3.2F,           56.14818F, 16.18867F,  3.1999998F,     61.5F,     16.0F,      3.1999998F,     118.5F,    16.0F,      3.1999998F,
    123.8518F, 16.18867F,  3.1999998F,     129.1771F, 16.753738F, 3.2F,           45.55068F, 17.692402F, 3.2F,           134.4493F, 17.692402F, 3.2F,
    0.0F,      19.0F,      3.2F,           40.35763F, 19.0F,      3.2F,           139.6424F, 19.0F,      3.2F,           180.0F,    19.0F,      3.2};
} // namespace
```
Should be const and in an anonymous namespace.

Reply:
Done